src/frontend/src/common/components/library/data_subview/components/DownloadModal/style.ts

```diff
@@ -111,6 +118,7 @@ export const DownloadType = styled.div`
 `;
 
 export const StyledButton = styled(Button)`
+  max-width: fit-content;
```

Comment:
Changing to flex made the button full-width, so this puts it back to the correct size.